### DISCUSSION TITLE
Repository pattern is possible with prisma client?

### DISCUSSION THREAD

**Discussion by fnoquiq** [prisma/prisma#5641] [Q&A] [answered]
Is there any way to make a mock in the prisma-client? I would like to apply the repository pattern...

**Comment by fnoquiq** [comment, marked as answer]
Yes, prisma is not related to repository pattern. I can't delete this post :/

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
